packages/core/src/domain/error/trackRuntimeError.spec.ts

```diff
@@ -315,3 +310,20 @@ describe('instrumentUnhandledRejection', () => {
     expect(stack).toBeDefined()
   })
 })
+
+function setupOnUnhandledrejectionSpy() {
+  if (!('onunhandledrejection' in window)) {
+    return null
+  }
```

Comment:
💭 **Thought**: I think it would be clearer to have this method always return a `jasmine.Spy`, and have a separated check in the `beforeEach`s. wdyt ?